{"event": "verbose", "uuid": "089927cf-2c9e-4fd8-b959-66c9184a82de", "counter": 3, "stdout": "\u001b[0;31mThe error appears to be in '/home/vescana/test_ansible_molecule/playbooks/site.yml': line 24, column 7, but may\u001b[0m", "start_line": 2, "end_line": 3, "runner_ident": "d1173030-8b16-4063-aaef-2bf196f839fd", "created": "2022-04-21T07:43:01.729769"}